spec/fluent.asdl

```diff
@@ -40,19 +44,16 @@ module Fluent
          | Placeable(expr expression, span? span)
 
     -- Expressions
-    expr = InlineExpression(inline_expr)
-         | BlockExpression(block_expr)
-    inline_expr = StringExpression(string)
```

Comment:
`inline_expr` is still used in line 70?

    arg = Expression(inline_expr)

Reply:
Thanks.